.github/workflows/publish.yml

```diff
@@ -39,6 +39,9 @@ jobs:
           - path: fractor-yaml
             repository: git@github.com:andreaswolf/fractor-yaml.git
             deploy-key: DEPLOY_KEY_YAML
+          - path: fractor-htaccess
+            repository: git@github.com:andreaswolf/fractor-htaccess.git
+            deploy-key: DEPLOY_KEY_HTACCESS
```

Comment:
@andreaswolf todo for you

Reply:
Done, key is added in the new repo (public key as deploy key) and here (private key in the secret var)